superset/utils/pandas_postprocessing.py

```diff
@@ -264,6 +264,17 @@ def pivot(  # pylint: disable=too-many-arguments
     #  Remove once/if support is added.
     aggfunc = {na.column: na.aggfunc for na in aggregate_funcs.values()}
 
+    # When dropna = False, the pivot_table function will calculate cartesian-product
+    # for MultiIndex.
+    # https://github.com/apache/superset/issues/15956
+    # https://github.com/pandas-dev/pandas/issues/18030
+    series_set = set()
+    to_string_list: Callable[[List[Any]], List[str]] = lambda lst: [str(_) for _ in lst]
```

Comment:
Could we even wrap the `"_".join` inside the lambda?

Reply:
Sure! thanks!